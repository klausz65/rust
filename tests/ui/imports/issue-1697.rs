// Testing that we don't fail abnormally after hitting the errors

use unresolved::*;
//~^ ERROR unresolved import `unresolved` [E0432]
//~| NOTE use of unresolved module or unlinked crate `unresolved`
//~| HELP if you wanted to use a crate named `unresolved`, use `cargo add unresolved` to add it to your `Cargo.toml`

fn main() {}
