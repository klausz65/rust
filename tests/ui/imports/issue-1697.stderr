error[E0432]: unresolved import `unresolved`
  --> $DIR/issue-1697.rs:3:5
   |
LL | use unresolved::*;
   |     ^^^^^^^^^^ use of unresolved module or unlinked crate `unresolved`
   |
help: if you wanted to use a crate named `unresolved`, use `cargo add unresolved` to add it to your `Cargo.toml` and import it in your code
   |
LL + extern crate unresolved;
   |

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0432`.
